x/kavadist/keeper/keeper_test.go

```diff
@@ -0,0 +1,37 @@
+package keeper_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/suite"
+
+	"github.com/kava-labs/kava/x/kavadist/testutil"
+	"github.com/kava-labs/kava/x/kavadist/types"
+)
+
+type keeperTestSuite struct {
+	testutil.Suite
+}
+
+func (suite *keeperTestSuite) SetupTest() {
+	suite.Suite.SetupTest()
+}
+
+func TestKeeperTestSuite(t *testing.T) {
+	suite.Run(t, new(keeperTestSuite))
+}
+
+func (suite *keeperTestSuite) TestGetPreviousBlock_NoPreviousBlock() {
+	blockTime, found := suite.Keeper.GetPreviousBlockTime(suite.Ctx)
+	suite.Require().False(found)
+	suite.Require().Equal(blockTime, types.DefaultPreviousBlockTime)
```

Comment:
nit: It doesn't look like the anything depends on `DefaultPreviousBlockTime` being returned when the time is not found in the store. I'd probably remove the check and align `keeper.GetPreviousBlockTime` with go conventions by returning the standard `time.Time{}`.